Rubberduck.Core/UI/Controls/PersistGroupExpandedStateBehavior.cs

```diff
@@ -44,15 +44,20 @@ protected override void OnAttached()
         {
             base.OnAttached();
 
-            var states = GetExpandedStateStore();
-            var expanded = !states.ContainsKey(GroupName ?? string.Empty) ? (bool?)null : states[GroupName ?? string.Empty];
+            // Ensure the visual tree is fully loaded before trying to access parents
+            AssociatedObject.Dispatcher.BeginInvoke(new System.Action(() =>
```

Comment:
Newing up an action delegate explicitly seems a bit old-fashioned, typically we'd simply pass the lambda `() => DoStuff()`. If there isn't a particular reason for this delegate to be explicitly created like this, it would be preferable to match the style of the rest of the code base.